index.html

```diff
@@ -137,12 +137,32 @@
     </div>
   </script>
 
+  <!-- Project List item -->
+  <script type="text/template" id="project-list-template">
```

Comment:
As per my note in my earlier review, please change "project-list-template" to "project-list-item-template".